app/Kconfig

```diff
@@ -9,34 +13,36 @@ config USB_DEVICE_PRODUCT
 config BT_DEVICE_NAME
 	default ZMK_KEYBOARD_NAME
 
-config ZMK_KSCAN_EVENT_QUEUE_SIZE
-	int "Size of the event queue for KSCAN events to buffer events"
-	default 4
+config USB_DEVICE_VID
+	default 0x1D50
+
+config USB_DEVICE_PID
+	default 0x615E
+
+config USB_DEVICE_MANUFACTURER
+	default "ZMK Project"
 
 menu "HID Output Types"
 
-menuconfig ZMK_USB
+config ZMK_USB
 	bool "USB"
 	select USB
 	select USB_DEVICE_STACK
 	select USB_DEVICE_HID
 
-if USB
+if ZMK_USB
```

Comment:
That makes sense. Switching to `ZMK_USB` allows it to be shown/hidden based on the selection of `USB` in menuconfig. I see the value of keeping it as `USB` though. Thinking through it, the Init Priorities (both USB and BLE) should probably be moved outside of basic keyboard setup since most users will not need to modify those defaults.

Reply:
Yeah, those init priorities really should be moved somewhere "advanced"